### DISCUSSION TITLE
BCC behaviour

### DISCUSSION THREAD

**Discussion by adamu** [swoosh/swoosh#772] [General]
What is the expected BCC behaviour of this library?

I'm asking because using the SMTP adapter at least, the `BCC` header is included in the delivered mail.

gen_smtp does not remove it: https://github.com/gen-smtp/gen_smtp/issues/11

I think the expected and good-citizen approach here is for the mail client (Swoosh in this case) to strip the Bcc header from the email, but still include those addresses in the list of recipients in the SMTP request.

**Comment by princemaple** [comment]
Interesting. I never used bcc with smtp and haven't used smtp in general in a long while. Happy to checkout some reference implementations in other languages and/or accept PR.

**Comment by adamu** [comment]
Just looking at the spec now: https://datatracker.ietf.org/doc/html/rfc2822#section-3.6.3

**Comment by princemaple** [comment]
Thanks heaps for bringing this up and sending the PR 💜
